src/filehandler/pemhandler.cpp

```diff
@@ -1,44 +1,43 @@
-#include "pemhandler.h"
-#include "utils/base64.h"
-#include "utils/hex.h"
+#include "filehandler/pemhandler.h"
+#include "base64.h"
+#include "hex.h"
 
 #include <fstream>
 #include <iostream>
 #include <sodium.h>
 
 namespace ih
 {
-PemInputHandler::PemInputHandler(wrapper::PemHandlerInputWrapper const inputWrapper) :
-        IFileHandler(inputWrapper.getPemFilePath()),
-        m_inputData(inputWrapper)
+PemFileReader::PemFileReader(std::string const filePath) :
```

Comment:
What does the namespace / abbreviation _ih_ stand for?

Reply:
ih = inputhandler. However, I intend to put everything inside SDK APIs into a namespace called `erd`
I left it there because I wanted to take care of namespaces in another PR.